### PR TITLE
ARP: Python.Python.3 version 3.5.1150

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23339)

### CODE REVIEW

File: manifests/p/Python/Python/3/3.5.1150/Python.Python.3.installer.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 PackageIdentifier: Python.Python.3
-PackageVersion: 3.5.1
+PackageVersion: 3.5.1150
```
Comment (ItzLevvie) [inline manifests/p/Python/Python/3/3.5.1150/Python.Python.3.installer.yaml:3]:
```suggestion
PackageVersion: 3.5.1150.0
```

File: manifests/p/Python/Python/3/3.5.1150/Python.Python.3.locale.en-US.yaml [hunk 2]
```diff
@@ -1,18 +1,18 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
 PackageIdentifier: Python.Python.3
-PackageVersion: 3.5.1
+PackageVersion: 3.5.1150
```
Comment (ItzLevvie) [inline manifests/p/Python/Python/3/3.5.1150/Python.Python.3.locale.en-US.yaml:3]:
```suggestion
PackageVersion: 3.5.1150.0
```

File: manifests/p/Python/Python/3/3.5.1150/Python.Python.3.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
 PackageIdentifier: Python.Python.3
-PackageVersion: 3.5.1
+PackageVersion: 3.5.1150
```
Comment (ItzLevvie) [inline manifests/p/Python/Python/3/3.5.1150/Python.Python.3.yaml:3]:
```suggestion
PackageVersion: 3.5.1150.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Merge conflicts.

**Review by jedieaston — APPROVED** [review]
image

I think @wingetbot needs a vacation.

**Pull request merged**